component/menu.go

```diff
@@ -179,7 +179,8 @@ func (m MenuItemStyle) Layout(gtx C) D {
 						return m.IconInset.Layout(gtx, func(gtx C) D {
 							iconSize := gtx.Px(m.IconSize)
 							gtx.Constraints = layout.Exact(image.Point{X: iconSize, Y: iconSize})
-							return m.Icon.Layout(gtx)
+							iconColor := color.NRGBA{A: 0xff}
```

Comment:
Instead of hard-coding a color, can we add a `IconColor` to the Style type and then populate it with `theme.Fg` in the constructor? That allows users of dark themes to make the icon light atop a dark background.

Reply:
Sounds good. Fixed now. Thanks